apps/expo-go/android/app/build.gradle

```diff
@@ -28,6 +28,20 @@ if (expoModulesCorePlugin.exists()) {
   applyKotlinExpoModulesCorePlugin()
 }
 
+apply plugin: 'com.facebook.react'
+
+react {
+  def projectRoot = rootDir.getAbsoluteFile().getParentFile().getAbsolutePath()
+
+  entryFile = file(["node", "-e", "require('expo/scripts/resolveAppEntry')", projectRoot, "android", "absolute"].execute(null, rootDir).text.trim())
+  reactNativeDir = new File(["node", "--print", "require.resolve('react-native/package.json')"].execute(null, rootDir).text.trim()).getParentFile().getAbsoluteFile()
+  hermesCommand = new File(["node", "--print", "require.resolve('react-native/package.json')"].execute(null, rootDir).text.trim()).getParentFile().getAbsolutePath() + "/sdks/hermesc/%OS-BIN%/hermesc"
+  codegenDir = new File("../../../../react-native-lab/react-native/packages/react-native-codegen")
+
+  cliFile = new File(["node", "--print", "require.resolve('@expo/cli')"].execute(null, rootDir).text.trim())
```

Comment:
these require.resolve are all risky and should use the package inside react-native-lab